crates/edr_napi/package.json

```diff
@@ -27,6 +27,10 @@
   "engines": {
     "node": ">= 18"
   },
+  "exports": {
+    ".": "./index.js",
+    "./solidity-tests": "./dist/src/ts/solidity_tests.js"
+  },
   "files": [
```

Comment:
I wonder why `Cargo.toml` and `build.rs` are here? I don't think they're necessary.